tmt/plugins/__init__.py

```diff
@@ -437,3 +445,40 @@ def __call__(self, logger: Logger) -> ModuleT:
 
         assert self._module  # narrow type
         return self._module
+
+
+def iter_plugin_registries() -> Iterator[PluginRegistry[Any]]:
+    # TODO: maybe there is a better way, but as of now, registries do
+    # not report their existence, there is no method to iterate over
+    # them. Using a static list for now.
+    from tmt.base import Plan, Story, Test
+    from tmt.checks import _CHECK_PLUGIN_REGISTRY
+    from tmt.frameworks import _FRAMEWORK_PLUGIN_REGISTRY
+    from tmt.package_managers import _PACKAGE_MANAGER_PLUGIN_REGISTRY
+    from tmt.plugins.plan_shapers import _PLAN_SHAPER_PLUGIN_REGISTRY
+    from tmt.steps.discover import DiscoverPlugin
+    from tmt.steps.execute import ExecutePlugin
+    from tmt.steps.finish import FinishPlugin
+    from tmt.steps.prepare import PreparePlugin
+    from tmt.steps.provision import ProvisionPlugin
+    from tmt.steps.report import ReportPlugin
```

Comment:
Seems the `prepare.feature` registry is missing?

Reply:
True, added in 484329b5.

Reply:
Thanks!